flambe/tokenizer/word.py

```diff
@@ -43,29 +49,49 @@ class NGramsTokenizer(Tokenizer):
     ngrams: Union[int, List[int]]
         An int or a list of ints. If it's a list of ints, all n-grams
         (for each int) will be considered in the tokenizer.
+    exclude_stopwords: bool
+        Whether to exlude stopword or not. See the related param stop_words
+    stop_words: Optional[List]
+        List of stop words to exclude when exclude_stopwords is True. If None
+        set to nltk.corpus.stopwords.
 
     """
-    def __init__(self, ngrams: Union[int, List[int]] = 1) -> None:
+    def __init__(self, ngrams: Union[int, List[int]] = 1,
+                 exclude_stopwords: bool = False,
+                 stop_words: Optional[List] = None) -> None:
         """[summary]
```

Comment:
Summary should be changed

Reply:
Docstring says: [summary] on the first line. I think you want to put "Initialize the NgramsTokenizer"